### PR TITLE
feat: nav component

### PR DESCRIPTION
**Pull request by HarshnaNHaswani**
image

### CODE REVIEW

File: pages/components/navigation.html [hunk 1]
```diff
@@ -107,7 +107,25 @@ <h1 clss="page-title">navigation</h1>
...
+
+  <nav class="nav nav-scrolling bg-secondary">
+    <ul class = "list list-inline">
+      <li class="list-item"> <a href=/>About </a> </li>
+      <li class="list-item"> <a href=/>Documentation </a> </li> 
+      <li class="list-item"> <a href=/>Installation </a> </li> 
+      <li class="list-item"> <a href="https://github.com/HarshnaNHaswani/Ira">GitHub</a></li>
+    </ul>
+  </nav>
+
```
Comment (mohitdhatrak) [inline pages/components/navigation.html:128]: why 2 navs? i can't understand as you have not added any screenshots to see.

**Review by mohitdhatrak — COMMENTED** [review]
Looks clean, good work! If possible try to provide deploy preview or screenshots of the changes so we can have a look too

File: pages/components/navigation.html (hunk 1 above)
Comment (Rishi05051997) [reply pages/components/navigation.html:128]: Is second nav for responsiveness initially you are hiding something like this

**Review by Rishi05051997 — COMMENTED** [review]
Please clarify doubt other things look good

File: pages/components/navigation.html (hunk 1 above)
Comment (HarshnaNHaswani) [reply pages/components/navigation.html:128]: Forgot to add, please check it now

**Comment by HarshnaNHaswani** [comment]
> Please clarify doubt other things look good

Hi, it is for the second type of navigation.

**Comment by HarshnaNHaswani** [comment]
@mohitdhatrak @Rishi05051997 thanks for your time and review. I apologize I forgot to add  a  screenshot earlier

**Pull request merged**